packages/generators/add-generator.ts

```diff
@@ -300,7 +308,7 @@ export default class AddGenerator extends Generator {
 										.pop()
 										.replace(".js", "")
 							)
-							.find((p: string): boolean => p.toLowerCase().indexOf(answerToAction.actionAnswer) >= 0);
```

Comment:
Don't remove the lowercase here, it's needed because if the user just types we will install the wrong plugin

Reply:
I found that this `toLowerCase` was one of the main issues, 
it will convert the `DefinePlugin` to `defineplugin` and then in `glob.sync` method..this plugin won't be addressed as in the directory its name as `DefinePlugin` only.

Reply:
it will check indexof, so if you start typing `define` it will convert to `DefinePlugin` since we're not assigning the lowercase. Important not to strict typecheck here bc of user experience. Ideally should have substring matching instead

Reply:
I didnt get that.
suppose we have this array 
```
let x = ["DefinePlugin", "ProgessPlugin"]
```
Now here are the output with/without `toLowerCase`
```bash
x.map(p => p.toLowerCase()).indexOf("DefinePlugin")
// -1
x.map(p => p).indexOf("DefinePlugin")
// 1
x.map(p => p.toLowerCase()).indexOf("defineplugin")
// 1
x.map(p => p).indexOf("defineplugin")
// -1
```
I guess we should replace it with regex , what say @evenstensberg